Client/trunk/ParaEngineClient/ParaScriptBindings/ParaScripting3.cpp

```diff
@@ -121,6 +121,8 @@ void CNPLScriptingState::LoadHAPI_SceneManager()
 
 				def("SetBlockUserData",&ParaTerrain::SetBlockUserData),
 				def("SetBlockUserDataByIdx",&ParaTerrain::SetBlockUserDataByIdx),
+
+				def("SetBlockVisible", &ParaTerrain::SetBlockVisible),
```

Comment:
shall we expose this api in RegisterBlockTemplate  instead? so that we can also change other properties in the same place.